kroxylicious/src/main/java/io/kroxylicious/proxy/config/Configuration.java

```diff
@@ -26,4 +26,10 @@ public List<MicrometerDefinition> getMicrometer() {
     public boolean isUseIoUring() {
         return useIoUring();
     }
+
+    public List<io.kroxylicious.proxy.model.VirtualCluster> validClusters() {
```

Comment:
Should this still be `validClusters` when it doesn't filter anything?  It sounds like it returns a subset of clusters.

Reply:
Its a good question 😁 

I added it so I had somewhere post config parsing to validate the cluster names were unique, and I still feel that it represents a valid and sensible hook to have. However I haven't figured out how to include the port de-conflictor here. Maybe I should just back it out entirely as we are moving to blowing up on invalid config.